.github/workflows/maintainer_merge_comment.yml

```diff
@@ -0,0 +1,42 @@
+name: Maintainer merge (comment)
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  ping_zulip:
+    name: Ping maintainers on Zulip
+    if: (github.event.issue.pull_request != 'null') && (startsWith(github.event.comment.body, 'maintainer merge') || contains(toJSON(github.event.comment.body), '\r\nmaintainer merge'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check whether user is part of mathlib-reviewers team
+        uses: TheModdingInquisition/actions-team-membership@v1.0
+        with:
+          organization: 'leanprover-community'
+          team: 'mathlib-reviewers' # required. The team to check for
+          token: ${{ secrets.MATHLIB_REVIEWERS_TEAM_KEY }} # required. Personal Access Token with the `read:org` permission
+          comment: 'You seem to not be authorized' # optional. A comment to post if the user is not part of the team.
+                                                  # This feature is only applicable in an issue (or PR) context
```

Comment:
I removed these lines from the other workflows since they run in a context where this action is apparently unable to post comments, but I left them here since it seems like it might still work. Maybe we should remove them for consistency though?

Reply:
Hmm, I think it's fine. It is probably also the most likely scenario in which someone might make a mistake.